fossunited/fossunited/doctype/institute/institute.json

```diff
@@ -35,11 +36,17 @@
    "fieldtype": "Link",
    "label": "Country",
    "options": "Country"
+  },
+  {
+   "fieldname": "institute_name",
+   "fieldtype": "Data",
+   "label": "Name",
+   "reqd": 1
```

Comment:
We do not need to explicitly have a field called `institute_name`.  The naming series for this doctype is done via a prompt to the user. So, Frappe provides a `name` field automatically (which is also a required field) at the time of creation!

Reply:
Fair enough